wherescape/connectors/hubspot/python_hubspot_check_missing_columns.py

```diff
@@ -0,0 +1,113 @@
+"""
+Script that compares the columns from the Hubspot api with the columns in the
+WhereScape load tables. It will load all missing columns in a load table and
+compares it to the ds tables. If new ones have appeared, it add them to the
+WhereScape warning log messages so they turn up in the daily report in Slack.
+"""
```

Comment:
I think the docstring should document that usage of this module presumes the existence of a load_table with certain columns. 

Even better would be a create_load_table() function that creates exactly the right load_table in the repository

Reply:
If you want to use this OPEN SOURCE module. there are quite a lot of assumptions / precoditions to be met: 
1. You need to create a load table in Wherescape with a prefix of length 5
2. You need to call create_metadata()
3. You need to call create a datastore table from that load table in Wherescape, with standard dss columns 
4. The name of ds_table needs to be the same as that of the load_table, but with a prefix of ds_
5. The ds_table must reside in a datastore schema


I think you need to explain that in this module docstring, or refer to a manual in the Google Drive in this module docstring, such that you can also include printscreens